.phan/config.php

```diff
@@ -45,6 +45,7 @@
         "vendor/psr",   // fix PhanUndeclaredClass for psr 
         "vendor/zendframework", //fix PhanUndeclaredClass zend 
         "vendor/guzzlehttp",  // fix PhanUndeclaredClass guzzlehttp
+        "vendor/smarty", // fix PhanUndeclaredClass Smarty
```

Comment:
@kongtiaowang I think you can just do `vendor/` and it should get all the subfolders

Reply:
@johnsaigle It will take more time to do the phan check. I can do it.